internal/controllers/synthetic_devices_controller_test.go

```diff
@@ -135,7 +157,7 @@ func (s *VirtualDevicesControllerTestSuite) TestGetVirtualDeviceMintingPayload_U
 
 func (s *VirtualDevicesControllerTestSuite) TestGetVirtualDeviceMintingPayload_NoEthereumAddressForUser() {
 	email := "some@email.com"
-	user := test.BuildGetUserGRPC(testUserID, &email, nil, &users.UserReferrer{})
+	user := test.BuildGetUserGRPC(mockUserID, &email, nil, &users.UserReferrer{})
```

Comment:
It's difficult to believe that this is the problem.

Is this the janky users API mock? Maybe we should use a "real" one, I wonder if stuff is sharing between sessions.